src/mutations/helper/use-mutation-fetch.ts

```diff
@@ -78,9 +72,10 @@ export function useMutationFetch() {
       const client = new GraphQLClient(endpoint, {
         headers: auth.current
           ? {
-              Authorization: `Bearer ${auth.current.token}`,
+              Authorization: `Bearer `,
```

Comment:
This can be removed now, right?

Reply:
Yes, we just have to synchronize with cloudflare (and api) first.